setup.py

```diff
@@ -54,7 +52,7 @@ def finalize_options(self):
 
     def run(self):
         try:
-            out = subprocess.check_output(["cmake", "--version"])
+            _ = subprocess.check_output(["cmake", "--version"])
```

Comment:
この部分は`cmake --version`を実行してみて、cmakeが存在するかどうかを確認していますね。
変数への代入は不要だと思います。
```suggestion
            subprocess.check_output(["cmake", "--version"])
```

Reply:
修正しました。